samples/hello-terminal/terminal-workflow/src/main/java/com/squareup/sample/helloterminal/terminalworkflow/KeyStroke.kt

```diff
@@ -40,6 +41,7 @@ data class KeyStroke(
   }
 }
 
+@ObsoleteCoroutinesApi
```

Comment:
Haha `Obsolete` co-existing with `Experimental`. Life comes at you fast!

Reply:
Nit: this should be opting into this annotation, not using it directly.